### PR TITLE
Updates to contributing.md and readme

### PR DESCRIPTION
**Pull request by keegansmith21**
Changes to some existing docs. These need updating as they're out of date with the current state of the repo. I also make reference the the platform's dependent repositories as they use the same systems.

Notes for post-merge:
- Rename develop branch to main
- Update the two dependent repositories with a contributing guide that references this one

### CODE REVIEW

**Comment by keegansmith21** [comment]
@jdddog Please have a look over the changes and current state of the changed docs. It would be good to get your input - please make any changes you think necessary

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -156,6 +152,13 @@ How to enable popular IDEs to run the unittests:
 * PyCharm: PyCharm supports test discovery and execution for the Python unittest framework. You may need to configure
 PyCharm to use the unittest framework. Click PyCharm > Preferences > Tools > Python Integrated Tools and under the 
 Testing heading, choose `Unittests` from the `Default test runner` dropdown.
+* VSCode: VSCode supports test discovery and execution for the Pytyhon unittest framework. Under the *Testing* panel, configure the tests to use the unittest framework and search for tests beginning with *test_**. 
+  
+A *.env* file will also need to be configured with the following variables set:
+* GOOGLE_APPLICATION_CREDENTIALS - The path to your Google Cloud Project credentials
+* TEST_GCP_PROJECT_ID - Your GCP Project ID
+* TEST_GCP_DATA_LOCATION - The location of your GCP Project
+* TEST_GCP_BUCKET_NAME - The name of your GCP testing bucket
```
Comment (alexmassen-hane) [inline CONTRIBUTING.md:161]: Some unit tests for OpenAlex requires AWS credentials to create buckets and transfer to GCS. You may need to briefly mention this or just link to a docs page.

Comment (keegansmith21) [reply CONTRIBUTING.md:161]: It's only relevant when changing those tests, but yeah I'll make mention of it

**Review by kathrynnapier — APPROVED** [review]
Thanks @keegansmith21 - changes look good to me. I've also added in a code of conduct file as well, that I'd like @jdddog to take a quick look at

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -156,6 +182,18 @@ How to enable popular IDEs to run the unittests:
 * PyCharm: PyCharm supports test discovery and execution for the Python unittest framework. You may need to configure
 PyCharm to use the unittest framework. Click PyCharm > Preferences > Tools > Python Integrated Tools and under the 
 Testing heading, choose `Unittests` from the `Default test runner` dropdown.
+* VSCode: VSCode supports test discovery and execution for the Pytyhon unittest framework. Under the *Testing* panel, configure the tests to use the unittest framework and search for tests beginning with *test_**.
```
Comment (jdddog) [inline CONTRIBUTING.md]:
```suggestion
* VSCode: VSCode supports test discovery and execution for the Python unittest framework. Under the *Testing* panel, configure the tests to use the unittest framework and search for tests beginning with *test_**. 
```

**Review by jdddog — APPROVED** [review]
Thanks Keegan and Kathryn, the changes look good to me.

**Pull request merged**